### DISCUSSION TITLE
parkingLocation with ID.4

### DISCUSSION THREAD

**Discussion by MattiJarvinen** [tillsteinbach/VWsFriend#61] [Q&A] [answered]
Does the parkingLocation work with ID.3? With ID.4 it doesn't seem to return any value.

**Comment by tillsteinbach** [comment, marked as answer]
The answer is a bit complicated. This is my theory:
The car is able to provide the parking position, but you need the capability to be enabled.
My guess is that Volkswagen will provide that with a separate WeConnect Plus contract for what they will ask for an additional payment. This extra package is currently not sold because there is no reason to buy it as long as the WeConnect ID App cannot make use of the additional data. If they would sell it now they would face massive complains about it not being useful.
Maybe an insider is reading this thread and knows more, or even what could be done to be able to use that additional package if it exists.
